docs/source/pages/tutorials/exclude_include_checks.rst

```diff
@@ -0,0 +1,175 @@
+
+In this run, the check ``mcn_provenance_witness_level_one_1`` mentioned above will not be run. After running the analyze command, we can view the data that Macaron has gathered about the ``micronaut-projects/micronaut-core`` repository at ``v4.3.10`` in an HTML report. Note that the result of the excluded check is not recorded in the Macaron HTML report (this also happens to JSON reports and the database).
+
+.. code-block:: shell
+
+  open output/reports/github_com/micronaut-projects/micronaut-core/micronaut-core.html
+
+.. _fig_exclude_provenance_withness_level_one:
+
+
+.. figure:: ../../_static/images/exclude_provenance_withness_level_one.png
+   :alt: Check results for ``micronaut-projects/micronaut-core v4.3.10 wit`` with witness provenance check excluded
+   :align: center
+
+
+------------------------------------------------
+Configuring exclude/include checks in ini config
+------------------------------------------------
+
+You can configure the exclusion and inclusion of checks through the ``[analysis.checks]`` section in the ``defaults.ini`` file. These are the defaults values:
+
+.. code-block:: ini
+
+    [analysis.checks]
+    # By default, we don't exclude any checks.
+    exclude =
+    # By default, we run all checks available.
+    include = *
+
+The ``exclude`` and ``include`` options accept a list of strings (terminated by new lines). Each element in that list can either be:
+
+* An ID of a check
+* A glob pattern (similar to patterns used in `glob <https://docs.python.org/3/library/glob.html>`_). When a glob pattern is given, all check IDs that match it will be included in the corresponding ``exclude`` or ``include`` list.
+
+The two lists of ``exclude`` and ``include`` check IDs obtained from your configuration can be defined as:
+
+* ``EXPLICIT_EXCLUDE`` = checks excluded from the user configuration.
+* ``EXPLICIT_INCLUDE`` = checks included from the user configuration.
+
+Checks in Macaron depend on each other. Therefore, when you want to exclude/include a check, you must be aware of how that will affect its transitive children/parents:
+
+* If a check is excluded, all of the children which can be transitively reached from it will be excluded.
+* If a check is included, all of the parents which can be transitively reached from it will be included.
+
+We define the list of check IDs after extending to transitive parents/children as:
+
+* ``EFFECTIVE_INCLUDED`` = ``EXPLICIT_INCLUDE`` set plus transitive parents
+* ``EFFECTIVE_EXCLUDED`` = ``EXPLICIT_EXCLUDE`` set plus transitive children
+
+The final list of checks that will run can be defined as:
+
+* ``FINAL_INCLUDED`` = ``EFFECTIVE_INCLUDED`` minus ``EFFECTIVE_EXCLUDED``
+
+-------------
+More examples
+-------------
+
+^^^^^^^^^^^^^^^^^^^^^
+Not running any check
+^^^^^^^^^^^^^^^^^^^^^
+The following configurations will result in no check run and Macaron will return on error.
```

Comment:
```suggestion
The following configurations will result in no check running and Macaron will return on error.
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/667/commits/f72fc8ed825545d5a193083b711cd6a0720a0465